microcosm_pubsub/dispatcher.py

```diff
@@ -69,7 +70,7 @@ def configure_sqs_message_dispatcher(graph):
     """
     try:
         sqs_message_handlers = graph.sqs_message_handlers
-    except AttributeError:
+    except NotBoundError:
```

Comment:
This was a bug: use of the daemon `schema_mappings` property wasn't working since failure to supply the `sqs_message_handlers` binding would raise an uncaught exception.